Ghiurutan V. Alexandru/Homework3/src/controller/Server.java

```diff
@@ -0,0 +1,34 @@
+package controller;
+
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import model.Task;
+
+public class Server implements Runnable {
+	private BlockingQueue<Task> queue;
+	private AtomicInteger nrOfActiveTasks;
+
+	public Server(int nrOfTasks) {
+		queue = new ArrayBlockingQueue<Task>(nrOfTasks);
+	}
+
+	public void addTask(Task task) {
+		queue.add(task);
+		nrOfActiveTasks.incrementAndGet();
+	}
+
```

Comment:
Here could be a little problem related to the fact that addTask method is outside the run method.Should I call it from inside the run method?